mathics/builtin/files_io/files.py

```diff
@@ -387,54 +392,75 @@ class Get(PrefixOperator):
     def eval(self, path, evaluation: Evaluation, options: dict):
         "Get[path_String, OptionsPattern[Get]]"
 
-        def check_options(options):
-            # Options
-            # TODO Proper error messages
+        def eval_inner(self, path, evaluation: Evaluation, options: dict):
```

Comment:
Thanks for noticing the bug and PR. 

the actual evaluation implementation of builtins (aside from basic parameter checking) nowadays we put into `mathics.eval.<name-of-module>`. Here, _name_of_module_ would be `files_io.files`. 

Are you up for creating this module under `mathics.eval` and moving `eval_inner()` there? 

This not only reduces the size of `mathics.builtin.files_io.files` but it also allows other routines that might want to use the evaluation primitives to do so in a faster way, and may allow use to write an instruction-based interpreter.  See also https://mathics-development-guide.readthedocs.io/en/latest/extending/code-overview/python-modules.html#id6

Reply:
done.